myhpom/templatetags/mmh_tags.py

```diff
@@ -0,0 +1,20 @@
+from django import template
+
+
+register = template.Library()
+
+
+SUPPORTED_STATE_ORGAN_DONOR_INFO_URLS = {
+    'NC': 'https://www.donatelifenc.org/register/',
+    'SC': 'https://www.donatelifesc.org/register/',
+}
+
+DEFAULT_ORGAN_DONOR_INFO_URL = 'https://www.donatelife.net/register/'
```

Comment:
I like this solution for its simplicity - wish we'd done that for those state specific links actually.